examples/quickstart/feature_repo/example.py

```diff
@@ -40,38 +48,99 @@
 )
 
 driver_stats_fs = FeatureService(
-    name="driver_activity",
-    features=[driver_hourly_stats_view]
+    name="driver_activity", features=[driver_hourly_stats_view]
+)
+
+driver_hourly_stats_push_source = PushSource(
+    name="driver_hourly_stats_push_source",
+    batch_source=driver_hourly_stats,
 )
 
+driver_hourly_stats_stream_view = FeatureView(
+    name="driver_hourly_stats_stream",
+    entities=[driver],
+    ttl=timedelta(seconds=8640000000),
+    schema=[
+        Field(name="driver_id", dtype=Int64),
+        Field(name="conv_rate", dtype=Float32),
+        Field(name="acc_rate", dtype=Float32),
+        Field(name="avg_daily_trips", dtype=Int64),
+        Field(name="created", dtype=UnixTimestamp),
+    ],
+    online=True,
+    source=driver_hourly_stats_push_source,
+    tags={"production": "True"},
+    owner="test2@gmail.com",
+)
+
+
 input_request = RequestSource(
     name="input_request",
     schema=[
         Field(name="int_val", dtype=Int64),
     ],
 )
 
-@on_demand_feature_view(    # noqa
+
+@on_demand_feature_view(  # noqa
     sources=[
         driver_hourly_stats_view,
         input_request,
     ],
     schema=[
         Field(name="output", dtype=Float64),
-        Field(name="time_since_last_date", dtype=Int64),
+        Field(name="seconds_since_last_created_date", dtype=Float64),
+        Field(name="days_since_last_created_date", dtype=Int64),
     ],
 )
 def transformed_conv_rate(inputs: pd.DataFrame) -> pd.DataFrame:
     df = pd.DataFrame()
-    df['output'] = inputs['conv_rate'] + inputs['int_val']
-    df['time_since_last_date'] = (datetime.now() - pd.to_datetime(inputs['created'])).dt.seconds
+    df["output"] = inputs["conv_rate"] + inputs["int_val"]
+    datedelta = pd.to_datetime(datetime.utcnow(), utc=True) - pd.to_datetime(
+        inputs["created"], utc=True
```

Comment:
why do you use `created` timestamp, shouldn't it be the `event_timestamp`?

Reply:
updated to do both in https://github.com/Affirm/feast/pull/13